microsoft-edge/devtools-guide-chromium/resources/index.md

```diff
@@ -20,145 +20,179 @@ ms.date: 05/04/2021
+## Browse resource files by file type in the Frames tree in the Application tool
 
-To group resources together based on their file type:
+One way to view webpage resources grouped by file type is by using the **Frames** tree in the **Application** tool:
 
-1. Select the **Application** tab.  The **Application** tool opens.  By default, the **Manifest** pane usually opens first.
+1. To open DevTools, right-click the webpage, and then select **Inspect**.  Or, press `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.
 
-   ![The Application tool.](../media/resources-application-mainfest-airhorner.msft.png)
+1. In DevTools, on the main toolbar, select the **Application** tab.  If that tab isn't visible, click the **More tabs** (![More tabs button](../media/more-tabs-icon-light-theme.png)) button, or else the **More Tools** (![The More Tools icon](../media/more-tools-icon-light-theme.png)) button.
 
-1. Scroll down to the **Frames** pane.
+1. Select the **Application** tab.  The **Application** tool opens and by default shows the **Manifest** pane first:
+   
+   ![The Application tool](../media/resources-application-mainfest-airhorner.msft.png)
 
-   ![The Frames pane.](../media/resources-application-mainfest-airhorner-frames-expanded.msft.png)
+1. Scroll down to the **Frames** pane and then expand the folder that you're interested in:
 
-1. Expand the sections you're interested in.
+   ![The Frames tree in the Application tool](../media/resources-application-mainfest-airhorner-frames-expanded.msft.png)
 
-1. Select a resource to view it.
+1. Select a resource to view it in the **Application** tool:
 
-   ![View a resource in the Application panel.](../media/resources-application-mainfest-airhorner-expanded-resources.msft.png)
+   ![Viewing a resource in the Application tool](../media/resources-application-mainfest-airhorner-expanded-resources.msft.png)
 
 
 <!-- ====================================================================== -->
-## Browse files by type in the Network panel
+## Browse resource files by file type by filtering in the Network tool
+
+In the **Network** tool, when you view the list of resource files that make up the current webpage, you can filter which type of resources to list, such as only showing `.css` or `.js` files.  For example, you can click **CSS** to filter and only display CSS files:
 
-See [Filter by resource type](../network/index.md#filter-by-resource-type).
+![Filtering to only list CSS resource files in the Network Log](../media/resources-network-resources-filter-css.msft.png)
 
-![Filter for CSS in the Network Log.](../media/resources-network-resources-filter-css.msft.png)
+The above screenshot shows `main.css`, which is part of the [Inspect Network Activity Demo](https://microsoftedge.github.io/Demos/network-tutorial/).  The source files for that page are at [MicrosoftEdge / Demos > network-tutorial](https://github.com/MicrosoftEdge/Demos/tree/main/network-tutorial).
 
```

Comment:
```suggestion

For more information, see [Filter by resource type](../network/index.md#filter-by-resource-type) in _Inspect network activity_.

```
Not sure if accidentally got omitted.  This link is present in the live article at bottom:
https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/resources/#browse-files-by-type-in-the-network-panel